components/dashboard/src/index.css

```diff
@@ -35,23 +35,29 @@
 
 @layer components {
     button {
-        @apply cursor-pointer px-4 py-2 my-auto bg-green-600 hover:bg-green-700 border-2 border-green-800 text-gray-100 text-sm font-medium rounded-md focus:outline-none focus:ring focus:border-blue-300;
+        @apply cursor-pointer px-4 py-2 my-auto bg-gitpod-kumquat-dark hover:bg-gitpod-kumquat-darker text-white text-sm font-medium rounded-md focus:outline-none focus:ring;
```

Comment:
**thought**: Height looks good also with `36px` although this will break the pattern of the `8px` grid. Negiotiable! Let's do it. 💭

Reply:
we could reduce it to 32px but that might be too small.